src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -358,10 +382,25 @@ public function testThrowExceptionIfDaysIsInvalid()
 
     public function testSetDataWithDifferentNegativeUTCTimezoneDateTime()
     {
-        date_default_timezone_set('Pacific/Tahiti');
+        $form = $this->factory->create('date', null, array(
+            'format' => \IntlDateFormatter::MEDIUM,
+            'model_timezone' => 'America/New_York',
+            'view_timezone' => 'Pacific/Tahiti',
+            'input' => 'string',
+            'widget' => 'single_text',
+        ));
 
+        $form->setData('2010-06-02');
+
+        $this->assertEquals('01.06.2010', $form->getViewData());
```

Comment:
That said it's hard to believe that anyone using these types has relied on date being shifted in certain scenarios. I'd rather treat it as a bug.

I think it's fine to bring the options back and ignore them. We shouldn't suggest in the readme they're supported though.